openlibrary/templates/type/page/view.html

```diff
@@ -21,7 +21,7 @@ <h1>$title</h1>
 
 $ text = format(page.body)
 
-<div id="contentBody">
+<div id="contentBody" class="type--page">
```

Comment:
We've been using bem to define components where the prefix is the name.

Type doesn't seem like  a good name for a component. Is there some way we could apply your css rule without introducing this class?

Reply:
Complexity of the selector meaning?